README.md

```diff
@@ -16,10 +16,24 @@ Android Engineer Interview Questions
 ## Android
  - How does Garbage collection works?
  - What is a dangling pointer?
+ - Explain the terms reference counting, mark and sweep? 
  - Elaborate Memory Leak?
  - Explain fragment Lifecycle when it comes to ViewPager and sliding between different fragments.
  - Difference between FragmentStateAdapter and FragmentStatePagerAdapter.
- - 
+ - Difference between Serializable and Parcelable? What are the disadvantages of Serializable?
+ - How you could implement observable SharedPrefs or observable Databases i.e. Observe a certain key/table/query?
+ - How does layout inflation work from xml tags to view reference in memory?
```

Comment:
Ditto

Reply:
This is the most common occurring question around data serialisation. I would want to keep it